hack/deployer/README.md

```diff
@@ -1,71 +1,53 @@
+### Provision
 
-## Typical usage
+* GKE
 
-While in repo root, run once:
-```
-make dependencies
-cd hack/deployer
-go build
-```
+  * Install [Google Cloud SDK](https://cloud.google.com/sdk/install)
+  * Install Google Cloud SDK beta components by running `gcloud components install beta`
+  * Make sure that container registry authentication is correctly configured as described [here](https://cloud.google.com/container-registry/docs/advanced-authentication)
+  * Set `GCLOUD_PROJECT` to the name of the GCloud project you wish to use
+  * Run from the [project root](/):
 
-Then, depending on the provider, run the following:
+    ```bash
+    make switch-gke bootstrap-cloud
+    ```
 
-For GKE, with your GCLOUD_PROJECT:
-```
-cat > config/deployer-config.yml << EOF
-id: gke-dev
-overrides:
-  clusterName: dkowalski-dev-cluster
-  gke:
-    gCloudProject: GCLOUD_PROJECT
-EOF
-```
+* AKS
 
-For AKS, with your ACR_NAME and RESOURCE_GROUP:
-```
-cat > config/deployer-config.yml << EOF
-id: aks-dev
-overrides:
-  clusterName: dkowalski-dev-cluster
-  aks:
-    resourceGroup: RESOURCE_GROUP
-    acrName: ACR_NAME
-EOF
-``` 
-
-
-Then, to create, run:
-```
-./deployer execute
-```
+  * Install [Azure CLI](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli?view=azure-cli-latest)
+  * Set `RESOURCE_GROUP` to the name of the Resource Group you wish to deploy in
+  * Set `ACR_NAME` to the name of Azure Container Registry you wish to use
+  * Run from the [project root](/):
 
-Then, to delete, run: 
+    ```bash
+    make switch-aks bootstrap-cloud
+    ```
+
+### Deprovision
+
+```bash
+make delete-cloud
 ```
-./deployer execute --operation delete
-``` 
 
+## Advanced usage
 
-## CI usage
+Deployer uses two config files:
 
-CI will populate deployer-config with vault login information and deployer will fetch the needed secrets. Secrets will differ depending on the provider chosen.
+* `config/plans.yml` - to store defaults/baseline settings for different use cases (different providers, CI/dev)
+* `config/deployer-config-*.yml` - to "pick" on of the predefined configs from config/plans.yml and allow overriding settings.
 
-## CI impersonation
+You can adjust many parameters that clusters are deployed with. Exhaustive list is defined in [settings.go](runner/settings.go).
 
-To facilitate testing locally, developers can run "as CI". While the credentials and vault login method are different it does allow fetching the same credentials and logging in as the same service account as CI would. This aims to shorten the dev cycle for CI related work and debugging.
+Running `make switch-*` will generate config/deployer-config-*.yml file for the respective provider using environment variables specific to that providers configuration needs. After the file is generated, you can make edit it to suit your needs and run `make bootstrap-cloud` to deploy. Currently chosen provider is stored in `config/provider` file.
```

Comment:
Maybe put config/deployer-config-*.yml between two ` to be consistent

Reply:
Good catch.